### PR TITLE
Small improvements to the Matter documentation

### PR DESCRIPTION
**Pull request by marcelveldt**
## Proposed change
- remove some outdated and/or redundant documentation
- add some instructions to run matter server on docker

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -83,25 +83,8 @@ Image taken from [this excellent article by The Verge](https://www.theverge.com/
...
 
-The only supported configuration (for now) for the Matter integration is by running the officially provided Home Assistant Matter add-on. Running the [Matter server](https://github.com/home-assistant-libs/python-matter-server) by any other means is at your own risk and is currently not officially supported.
+The recommended configuration for the Matter integration is by running the officially provided Home Assistant Matter add-on on Home Assistant OS, due to various requirements to the host OS and configuration. For Homeassistant Container based installs, we also provide a docker image which you can run, see for instructions the repository of the [Matter server](https://github.com/home-assistant-libs/python-matter-server) including all requirements and instructions for your host setup. It is not possible to run Matter on Home Assistant Core only based installs.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
### Supported installation types

It is recommended to run the Matter add-on on Home Assistant OS. This is currently the best-supported option. 

If you run Home Assistant in a container, you can run a docker image of the [Matter server](https://github.com/home-assistant-libs/python-matter-server). The requirements and instructions for your host setup are described on that github page.

Running Matter on a Home Assistant Core installation is not supported.
```

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
-### Nanoleaf Matter bulbs and Lightstrips
+### Nanoleaf (Essentials) Matter bulbs and Lightstrips
 
 - Although the products work great once commissioned, multiple users have reported that commissioning them can be a bit difficult and requires some patience and multiple resets or optimizations to your home network.
 - Check the [Nanoleaf Matter infopage](https://nanoleaf.me/en-EU/integration/matter/) for all supported products and instructions.
 
 ### Philips Hue (V2) Bridge
 
-The Philips Hue V2 bridge supports Matter since a recent update (the beta program closed, it is now officially available). You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
+The Philips Hue V2 bridge supports Matter since a recent update. You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
The Philips Hue V2 bridge supports Matter. You can enable Matter support in the Hue app. You can then commission it to Home Assistant and other fabrics.
```

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
 
 - Binding the Hue bridge to Home Assistant does not make sense because you will lose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
 - You will need a Hue/Signify (cloud) account and the app before you can use Matter.
 - Device events for example for dimmer remotes are not supported.
 - Only basic control of lights is supported, no scenes, events, effects etc.
 
+### Switchbot Hub 2
+
+Switchbot has released a (beta) firmware update to enable Matter support on their Hub 2, effectively bridging (some of the) devices on the Hub 2, such as curtain motors into Matter. The Switchbot Hub 2 is a Matter bridge device.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
SwitchBot has released a (beta) firmware update to enable Matter support on their Hub 2.  The SwitchBot Hub 2 is a Matter bridge device. It is bridging some of the devices, such as curtain motors, into Matter.
```

File: source/_integrations/matter.markdown [hunk 4]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
 - Binding the Hue bridge to Home Assistant does not make sense because you will lose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
 - You will need a Hue/Signify (cloud) account and the app before you can use Matter.
 - Device events for example for dimmer remotes are not supported.
 - Only basic control of lights is supported, no scenes, events, effects etc.
 
+### Switchbot Hub 2
+
+Switchbot has released a (beta) firmware update to enable Matter support on their Hub 2, effectively bridging (some of the) devices on the Hub 2, such as curtain motors into Matter. The Switchbot Hub 2 is a Matter bridge device. 
+
+- You need to enable the Matter bridge support in the Switchbot app, copy the code and use that to commission the Hub to HA (or use a second device to scan the QR code).
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- To use Matter, in the SwitchBot app, enable Matter bridge support. Then, copy the code and use that to commission the Hub to Home Assistant. Another option is to use a second device to scan the QR code.
```

File: source/_integrations/matter.markdown [hunk 5]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
 
-The Philips Hue V2 bridge supports Matter since a recent update (the beta program closed, it is now officially available). You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
+The Philips Hue V2 bridge supports Matter since a recent update. You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
 
 - Binding the Hue bridge to Home Assistant does not make sense because you will lose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
 - You will need a Hue/Signify (cloud) account and the app before you can use Matter.
 - Device events for example for dimmer remotes are not supported.
 - Only basic control of lights is supported, no scenes, events, effects etc.
 
+### Switchbot Hub 2
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
### SwitchBot Hub 2
```

File: source/_integrations/matter.markdown [hunk 6]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
+
+Switchbot has released a (beta) firmware update to enable Matter support on their Hub 2, effectively bridging (some of the) devices on the Hub 2, such as curtain motors into Matter. The Switchbot Hub 2 is a Matter bridge device. 
+
+- You need to enable the Matter bridge support in the Switchbot app, copy the code and use that to commission the Hub to HA (or use a second device to scan the QR code).
+- Device support is limited, you bridge specific devices to Matter by adding them as "Secondary device" in the app. Note that not all switchbot devices can be bridged.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Device support is limited. You bridge specific devices to Matter by adding them as **Secondary device** in the app. Note that not all SwitchBot devices can be bridged.
```

File: source/_integrations/matter.markdown [hunk 7]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
+- Enabling Matter support does not convert the actual swiotchbot devices into matter devices, those still need to be within the (bluetooth) range of the Hub.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Enabling Matter support does not convert the actual SwitchBot devices into matter devices. Those still need to be within the Bluetooth range of the hub.
```

File: source/_integrations/matter.markdown [hunk 8]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
+- Bridged Switchbot devices show up with a rather weird/technical name in Home Assistant. This is a known issue we've reported to Switchbot.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Bridged SwitchBot devices appear with a rather technical name in Home Assistant. This is a known issue.
```

File: source/_integrations/matter.markdown [hunk 9]
```diff
@@ -209,27 +189,37 @@ Did you test a device that is not listed below? It would be greatly appreciated
...
+
+
 ### Tasmota
 
 Tasmota supports Matter over IP on all ESP32 based devices (in experimental phase). Follow the [instructions](https://tasmota.github.io/docs/Matter/).
 
 ### TP-Link Tapo P125M (power plug)
 
-- Look for the M addition in the model name, a device without the M (regular P125) is not Matter compliant.
+- TP-Link Tapo P125MLook for the M addition in the model name, a device without the M (regular P125) is not Matter compliant.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Look for the _M_ addition in the model name. A device without the M (regular P125) is not Matter compliant.
```

File: source/_integrations/matter.markdown [hunk 10]
```diff
@@ -238,15 +228,18 @@ Tasmota supports Matter over IP on all ESP32 based devices (in experimental phas
 
 ### General recommendations
 
-- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk.
+- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk but we do provide tips and tricks how to run the Matter Server as docker container yourself, including all requirements to the host and networking on the [matter server repository](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md)
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS version 10 and above. Not using Home Assistant OS is at your own risk. We do provide some [documentation](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md) on how to run the Matter Server as a docker container. The documentation includes a description of the host and networking requirements. 
```

File: source/_integrations/matter.markdown [hunk 11]
```diff
@@ -238,15 +228,18 @@ Tasmota supports Matter over IP on all ESP32 based devices (in experimental phas
...
 
-- Start simple and work from there, keep your network easy and add for example an ESP32 test device. Once that works, move on to the next step or more devices.
+- Start simple and work from there, keep your network simple and add for example an ESP32 test device. Once that works, move on to the next step or more devices.
 
 - Realize that you are an early adopter, both on the hardware side and on the software (controller) side so you may run into compatibility issues or features that are still missing. Report any issues you may find and help out others if you find a workaround or tested a device.
 
-- Make sure IPv6 (multicast) traffic travels freely from your network to the Home Assistant host. There is no requirement to have an IPv6-enabled internet connection or DHCPv6 server. However, IPv6 support has to be enabled (it's enabled by default on Home Assistant OS).
+- Make sure IPv6 (multicast) traffic travels freely from your network to the Home Assistant host. There is no requirement to have an IPv6-enabled internet connection or DHCPv6 server. However, IPv6 support has to be enabled (it's enabled by default on Home Assistant OS).
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Make sure IPv6 (multicast) traffic travels freely from your network to the Home Assistant host. There is no requirement to have an IPv6-enabled internet connection or DHCPv6 server. However, IPv6 support has to be enabled on Home Assistant. Go to **{% my network title="Settings > System > Network" %}**, and make sure **IPv6** is set to **Automatic** or **static**, depending on your network setup. If you're unsure, use **Automatic**.
```

File: source/_integrations/matter.markdown [hunk 12]
```diff
@@ -238,15 +228,18 @@ Tasmota supports Matter over IP on all ESP32 based devices (in experimental phas
...
+- Start simple and work from there, keep your network simple and add for example an ESP32 test device. Once that works, move on to the next step or more devices.
 
 - Realize that you are an early adopter, both on the hardware side and on the software (controller) side so you may run into compatibility issues or features that are still missing. Report any issues you may find and help out others if you find a workaround or tested a device.
 
-- Make sure IPv6 (multicast) traffic travels freely from your network to the Home Assistant host. There is no requirement to have an IPv6-enabled internet connection or DHCPv6 server. However, IPv6 support has to be enabled (it's enabled by default on Home Assistant OS).
+- Make sure IPv6 (multicast) traffic travels freely from your network to the Home Assistant host. There is no requirement to have an IPv6-enabled internet connection or DHCPv6 server. However, IPv6 support has to be enabled (it's enabled by default on Home Assistant OS). 
+
+- See more advanced recommendations to host and network configuration in the [readme of the matter server repository](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md).
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- For more detailed information on network configuration, refer to the [README of the Matter server repository](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md).
```

File: source/_integrations/matter.markdown [hunk 13]
```diff
@@ -266,4 +259,4 @@ Also see this [extended troubleshooting guide](https://developers.home.google.co
...
+Investigate your network topology if you experience any issues with discovering devices (like the initial commission keeps failing) or if devices become unavailable randomly. For instance, a setting on your router or Wi-Fi access point to "optimize" multicast traffic can harm the (discovery) traffic from Matter devices. Keep this in mind when you experience issues trying to add or control Matter devices. Protocols like Matter are designed for regular residential network setups and do not play nicely with enterprise networking solutions like VLANs, Multicast filtering, and (malfunctioning) IGMP snooping. Try to keep your network as simple and flat as possible to avoid issues.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
If you experience any issues with discovering devices (for example, if the initial commission keeps failing or if devices become unavailable randomly), investigate your network topology. For instance, a setting on your router or Wi-Fi access point to "optimize" multicast traffic can harm the (discovery) traffic from Matter devices. Keep this in mind when you experience issues trying to add or control Matter devices. Protocols like Matter are designed for regular residential network setups and may not integrate well with enterprise networking solutions like VLANs, Multicast filtering, and (malfunctioning) IGMP snooping. To avoid issues, try to keep your network topology as simple and flat as possible.
```

**Review by c0ffeeca7 — COMMENTED** [review]
Some suggestions. It might be a bit of heavy editing in some parts. Feel free to rephrase.

File: source/_integrations/matter.markdown (hunk 10 above)
Comment (marcelveldt) [reply source/_integrations/matter.markdown]: well, a container install is supported since this month but with all precautions and warnings applied

Comment (c0ffeeca7) [reply source/_integrations/matter.markdown]: Right. Let me rephrase. I hope it is more accurate now.

**Comment by marcelveldt** [comment]
Thanks @c0ffeeca7 !

File: source/_integrations/matter.markdown [hunk 14]
```diff
@@ -83,25 +83,14 @@ Image taken from [this excellent article by The Verge](https://www.theverge.com/
...
 
-The only supported configuration (for now) for the Matter integration is by running the officially provided Home Assistant Matter add-on. Running the [Matter server](https://github.com/home-assistant-libs/python-matter-server) by any other means is at your own risk and is currently not officially supported.
+### Supported installation types
 
-## Current state of the integration
+It is recommended to run the Matter add-on on Home Assistant OS. This is currently the best-supported option. 
 
-While the support for Matter is evolving, we will regularly update the Matter integration with new features or device support. Because it might be hard to track what's supported and what's not, we list the current state here and try to update this information regularly.
+If you run Home Assistant in a container, you can run a docker image of the [Matter server](https://github.com/home-assistant-libs/python-matter-server). The requirements and instructions for your host setup are described on that github page.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
If you run Home Assistant in a container, you can run a Docker image of the [Matter server](https://github.com/home-assistant-libs/python-matter-server). The requirements and instructions for your host setup are described on that GitHub page.
```

File: source/_integrations/matter.markdown [hunk 15]
```diff
@@ -238,15 +234,18 @@ Tasmota supports Matter over IP on all ESP32 based devices (in experimental phas
 
 ### General recommendations
 
-- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk.
+- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS version 10 and above. Not using Home Assistant OS is at your own risk. We do provide some [documentation](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md) on how to run the Matter Server as a docker container. The documentation includes a description of the host and networking requirements.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS version 10 and above. Not using Home Assistant OS is at your own risk. We do provide some [documentation](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md) on how to run the Matter Server as a Docker container. The documentation includes a description of the host and networking requirements. 
```

**Review by c0ffeeca7 — COMMENTED** [review]
Fix terminology issues I introduced with my review comments

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @marcelveldt  :+1:
